src/KubeLibrary/KubeLibrary.py

```diff
@@ -323,6 +340,14 @@ def filter_deployments_names(self, deployments):
         """
         return [d.metadata.name for d in deployments]
 
+    def filter_replicasets_names(self, replicasets):
+        """Filter deployment  names for list of deployments .
```

Comment:
Please update the documentation here from `deployments` to `replicasets`
You also need to regenerate the docs then.

Reply:
@Nilsty Done